app/lib/theme/widgets/SelectWithNotifyWidget.tsx

```diff
@@ -32,8 +41,13 @@ const SelectWithNotifyWidget: React.FunctionComponent<WidgetProps> = (
 
   const [updatePendingActionsFrom, isUpdatingPendingActionsFrom] =
     useUpdatePendingActionsFrom();
-  const revisionId = formContext.revisionId;
-  const revisionStatus = formContext.revisionStatus;
+  const projectRevision = formContext.projectRevision;
+  const query = useFragment(SelectWithNotifyWidgetFragment, projectRevision);
+  const [revisionId, revisionStatus, pendingActionsFrom] = [
+    query.id,
+    query.revisionStatus,
+    query.pendingActionsFrom,
+  ];
```

Comment:
You can rename variables when destructuring, makes the code much simpler
```suggestion
  const { id: revisionId, revisionStatus, pendingActionsFrom } = useFragment(SelectWithNotifyWidgetFragment, projectRevision);
```